crates/binding_core_node/Cargo.toml

```diff
@@ -4,7 +4,7 @@ build = "build.rs"
 edition = "2021"
 exclude = ["artifacts.json", "index.node"]
 license = "Apache-2.0"
-name = "node"
+name = "binding_core_node"
```

Comment:
nit: do we need `core` in the name? `bindings_node`, or `bindings_wasm` would work?

Reply:
Not required, but I selected it because `@swc/css` can use `bindings_css_node` and `bindings_css_wasm`

Reply:
it was just naming related questions, no major opinions since in any case actual binding for the npm pkg won't use this name.